File: php/classes/Baju.php
<?php

require_once('Aksesoris.php');

class Baju extends Aksesoris {
    private $untuk;
    private $size;
    private $merk;

    public function __construct($untuk = "", $size = "", $merk = "") {
        parent::__construct();
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    // Getter Methods
    public function getUntuk() {
        return $this->untuk;
    }

    public function getSize() {
        return $this->size;
    }

    public function getMerk() {
        return $this->merk;
    }

    // Setter Methods
    public function setUntuk($untuk) {
        $this->untuk = $untuk;
        return $this;
    }

    public function setSize($size) {
        $this->size = $size;
        return $this;
    }

    public function setMerk($merk) {
        $this->merk = $merk;
        return $this;
    }
}
?>

File: cpp/Baju.cpp
#include <iostream>
#include <string>
#include <list>
#include "Aksesoris.cpp"

using namespace std;

class Baju: public Aksesoris
{
    private:
        string untuk;
        string size;
        string merk;
    public:

        Baju()
        {
            this->untuk = "";
            this->size = "";
            this->merk = "";
        }

        string get_untuk() const// membaca nama
        {
            return this->untuk;
        }
        
        void set_untuk(string untuk) // mengubah nama
        {
            this->untuk = untuk;
        }

        string get_size() const// membaca size
        {
            return this->size;
        }
        
        void set_size(string size) // mengubah size
        {
            this->size = size;
        }

        string get_merk() const// membaca merk
        {
            return this->merk;
        }
        
        void set_merk(string merk) // mengubah merk
        {
            this->merk = merk;
        }

        ~Baju()
        {
            
        }
};